smdebug/core/hook.py

```diff
@@ -222,6 +225,12 @@ def __init__(
         self.mode_steps = {ModeKeys.GLOBAL: init_step}
         self.writer = None
 
+        # TODO: kannanva: Don't access env var here. Make timeline writer implementation
+        # similar to tb writer.
+        self.timeline_writer = TimelineWriter(
+            os.getenv(SM_PROFILER_FILE_PATH_ENV_STR, DEFAULT_SAGEMAKER_PROFILER_PATH)
+        )
```

Comment:
probably could create TimelineWriter within FileWriter like the rest of the writers. But timelinewriter doesn't use the functions of FileWriter